src/contracts/core/DelegationManager.sol

```diff
@@ -866,65 +862,68 @@ contract DelegationManager is
-    function operatorShares(address operator, IStrategy strategy) public view returns (uint256) {
-        uint64 totalMagnitude = allocationManager.getTotalMagnitude(operator, strategy);
-        return operatorDelegatedShares[operator][strategy].toOwnedShares(totalMagnitude).unwrap();
-    }
-
     /**
      * @notice Given a staker and a set of strategies, return the shares they can queue for withdrawal.
      * This value depends on which operator the staker is delegated to.
      * The shares amount returned is the actual amount of Strategy shares the staker would receive (subject
      * to each strategy's underlying shares to token ratio).
      */
-    function getDelegatableShares(
+    function getWithdrawableShares(
         address staker,
         IStrategy[] memory strategies
-    ) public view returns (OwnedShares[] memory ownedShares) {
+    ) public view returns (uint256[] memory withdrawableShares) {
         address operator = delegatedTo[staker];
         for (uint256 i = 0; i < strategies.length; ++i) {
             IShareManager shareManager = _getShareManager(strategies[i]);
             // TODO: batch call for strategyManager shares?
             // 1. read strategy deposit shares
             
             // forgefmt: disable-next-item
-            Shares shares = shareManager.stakerStrategyShares(staker, strategies[i]);
+            uint256 depositShares = shareManager.stakerDepositShares(staker, strategies[i]);
 
             // 2. if the staker is delegated, actual withdrawable shares can be different from what is stored
             // in the StrategyManager/EigenPodManager because they could have been slashed
             if (operator != address(0)) {
                 uint64 totalMagnitude = allocationManager.getTotalMagnitude(operator, strategies[i]);
                 
                 // forgefmt: disable-next-item
-                ownedShares[i] = shares
-                    .toDelegatedShares(stakerScalingFactors[staker][strategies[i]])
-                    .toOwnedShares(totalMagnitude);
+                withdrawableShares[i] = SlashingLib.depositSharesToShares(
+                    depositShares,
+                    stakerDepositScalingFactor[staker][strategies[i]],
+                    totalMagnitude
+                );
+            } else {
+                withdrawableShares[i] = depositShares;
             }
         }
-        return ownedShares;
+        return withdrawableShares;
     }
 
     /**
-     * @notice Returns the number of actively-delegatable shares a staker has across all strategies.
-     * @dev Returns two empty arrays in the case that the Staker has no actively-delegateable shares.
+     * @notice Returns the number of shares in storage for a staker and all their strategies
+     * TODO: make cleaner, get rid of assembly
      */
-    function getDelegatableShares(
+    function getDepositedShares(
```

Comment:
why are we changing the semantic meaning here. this should be shares, not depositShares

Reply:
ah i see what you did. this is fine